src/api.ts

```diff
@@ -63,37 +330,157 @@ export default class VoucheryIO {
 
       throw new Error(errorMsg);
     }
+    return method === 'DELETE'
+      ? { status: request.status }
+      : turnUnderLineToCamelCase(request.data);
+  }
 
-    return request.data;
+  async createMainCampaign(
+    request: CreateMainCampaignRequest
+  ): Promise<CreateMainCampaignResponse> {
+    const requestBody = turnCamelCaseIntoUnderLine(request);
+    console.log('requestBody', requestBody);
+    return this.apiRequest({
+      url: '/campaigns',
+      method: 'POST',
+      data: JSON.stringify(requestBody),
```

Comment:
Why you need to do JSON.stringify for the data?
Axios will automatically do the transformation for object